qiling/debugger/qdb/core.py

```diff
@@ -203,6 +211,21 @@ def do_examine(self, args):
 
         examine_mem(self._ql, _xaddr, _count)
 
+    def do_context(self, *args):
+        """
+        show context information for current location
+        """
+        context_reg(self._ql, self._saved_states)
+        context_asm(self._ql, self._ql.reg.arch_pc, 4)
+
+    def do_show(self, *args):
+        """
+        show some runtime informations
+        """
+        self._ql.mem.show_mapinfo()
+        print("Qdb:", [(hex(idx), val) for idx, val in self.breakpoints.items()])
+        print("internal:", [(hex(idx), val) for idx, val in self._ql._addr_hook.items()])
```

Comment:
i am not too sure. do we need print or we can use nprint. just in case

Reply:
this information is for debugging qdb and qiling itself mostly , and use native print for not messing up log file